src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/General/MaintenanceType.php

```diff
@@ -96,4 +121,9 @@ public function getBlockPrefix()
     {
         return 'maintenance_general_block';
     }
+
+    public function getParent(): string
+    {
+        return ConfigurationType::class;
+    }
```

Comment:
To make a configuration form "multistore ready" (adding the multistore checkboxes), you have to apply the form extension, like this:

 ```php
public function getParent(): string
{
    return ConfigurationType::class;
}
`

Reply:
For now the checkbox are added in the maintenance configuration page template, but it will be made generic in configuration form templates, apparently the form templates are being reworked, I'm not sure how to it yet

Reply:
Can we improve naming a bit here?

> To make a configuration form "multistore ready" (adding the multistore checkboxes), you have to apply the form extension, like this:

and we're adding method `getParent` with `ConfigurationType::class`, it's not logical to me

Reply:
Well the `getParent` method is a Symfony convention, we can't rename that. 

I could rename the ConfigurationType class... the idea is that it's an extension for configuration forms only, but it's open for suggestions...

Reply:
I had the idea that this form extension could later include other things related to configuration forms (not only multistore related things). But I don't mind renaming it into `MultistoreConfigurationType`

Reply:
Yes, I know about `getParent` but extension name could be improved, `MultistoreConfigurationType` 👍